docs/docs/integrations/chat/azureml_chat_endpoint.ipynb

```diff
@@ -137,7 +137,7 @@
     "    endpoint_url=\"https://<your-endpoint>.<your_region>.inference.ml.azure.com/v1/chat/completions\",\n",
     "    endpoint_api_type=AzureMLEndpointApiType.serverless,\n",
     "    endpoint_api_key=\"my-api-key\",\n",
-    "    content_formatter=LlamaChatContentFormatter,\n",
+    "    content_formatter=OpenAIStyleChatContentFormatter,\n",
```

Comment:
Is "style" the right word? Can we see if there is a better naming concensus in the ecosystem? For instance, `litellm` uses the name `CustomOpenAI`.